src/App.js

```diff
@@ -1,24 +1,34 @@
-import React, { Component } from 'react';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import "./grid.css";
+import Search from "./Components/Search";
+import CurrentWeather from "./Components/CurrentWeather";
+import Upcoming from "./Components/Upcoming";
 
-class App extends Component {
 
+class App extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-    };
+    this.state = {};
   }
 
-  render () {
+render() {
     return (
       <div className="app">
-        <header className="app__header">
-        </header>
-        <main className="app__main">
-        </main>
+        <div className="container">
+          <Search />
+          <main className="app__main">
+            <CurrentWeather />
+            <Upcoming />
+          </main>
+        </div>
       </div>
     );
   }
 }
 
 export default App;
+
```

Comment:
don't keep old code around commented out.  it's a burden on the reader and, eventually, on you, too.

Reply:
noted